.storybook/preview-head.html

```diff
@@ -0,0 +1,3 @@
+<!-- .storybook/preview-head.html -->
+
+<link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.1/css/all.min.css">
```

Comment:
This is interesting to me.

Does it mean in production we are also loading those styles from cdn?

When I used `font-awesome` icons last time I remember, React version was including the styles by itself (or at the very least they were coming from `node_modules` package, not cdn).

Reply:
In production we don't need to do that this way, but icons in storybook here and in elements don't work (at least for me) so I added this which helped. Though I didn't put much attention to that, it could probably be done differently, I just wanted to be able to test the pr

Reply:
I know the FA situation is still weird. 

Currently `ui-kit` doesn't contain FontAwesome itself but documents it as a dependency to be installed by its users. What @mallachari did here was what we usually do in storybooks to satisfy such deps, in SaaS production we install from `node_modules` as you said.

One reason you don't want to be installing FA in a library as a regular dep is that many sites already use it (mostly using some CDN), and bundling it would result in duplication and possible version conflicts. We can discuss a better FA strategy on Slack if you have ideas, this is of course by no means perfect.